src/lib/converter/converter.ts

```diff
@@ -278,13 +296,27 @@ export class Converter extends ChildableComponent<
             );
         }
 
+        for (const packageProgram of packages) {
+            // This should not be undefined as we're just pulling
+            // the sourceFile from its associated program
+            const sourceFile = packageProgram.program.getSourceFile(
+                packageProgram.entryPoint
+            )!;
+            entries.push({
+                entryName: packageProgram.packageName,
+                file: packageProgram.entryPoint,
+                sourceFile,
+                program: packageProgram.program,
+            });
+        }
+
         for (const entry of entries) {
             context.setActiveProgram(entry.program);
             entry.context = this.convertExports(
                 context,
                 entry.sourceFile,
-                entryPoints,
-                getModuleName(resolve(entry.file), baseDir)
+                entries.length === 1,
```

Comment:
Moving `entries.length === 1` up from below wasn't necessary, I had some potential changes in this code which I think I didn't need in the end, but left this as I found it simpler to understand than seeing the whole array passed.